proxy/src/bin/proxy.rs

```diff
@@ -229,6 +227,10 @@ struct ProxyCliArgs {
     /// Configure if this is a private access proxy for the POC: In that case the proxy will ignore the IP allowlist
     #[clap(long, default_value_t = false, value_parser = clap::builder::BoolishValueParser::new(), action = clap::ArgAction::Set)]
     is_private_access_proxy: bool,
+
+    /// Configure whether all incoming requests have a Proxy Protocol V2 packet.
+    #[clap(value_enum, long, default_value_t = ProxyProtocolV2::Supported)]
```

Comment:
`Supported` is probably the best choice right now, but we should set defaults to what's usually expected in prod. Maybe add a TODO to change this to `Required`.